packages/h5wasm/src/__snapshots__/h5wasm-api.test.ts.snap

```diff
@@ -460,13 +460,13 @@ exports[`test file matches snapshot 1`] = `
       "type": 1,
       "vlen": false,
     },
-    "shape": [],
+    "shape": null,
     "type": {
       "class": "Float",
       "endianness": "little-endian",
       "size": 32,
     },
-    "value": undefined,
+    "value": null,
```

Comment:
The main change: empty datasets/attributes now have `null` shapes and values, just like in h5grove. They no longer appear as "Scalar" in the metadata viewer.